src/loader.ts

```diff
@@ -241,6 +249,45 @@ interface ModuleDefinitionArguments extends Array<any> {
 		return has;
 	})();
 
+	let listenerQueues = {};
+
+	const makeSignalError = function (message: string, info: ObjectMap): LoaderError {
+		return mix<LoaderError>(new Error(message), {
+			src: 'dojo/loader',
+			info
+		});
+	};
+
+	const signal = function(type: SignalType, args: {}): number | boolean {
```

Comment:
Can we call this `emit`? It would match what most JS event libraries are calling this functionality these days.